Firestore/Source/Local/FSTMemoryQueryCache.mm

```diff
@@ -38,11 +38,13 @@ @interface FSTMemoryQueryCache ()
 
 @property(nonatomic, assign) FSTListenSequenceNumber highestListenSequenceNumber;
 
+@property(nonatomic, strong) FSTSnapshotVersion *lastRemoteSnapshotVersion;
+
 @end
 
 @implementation FSTMemoryQueryCache {
   /** The last received snapshot version. */
-  FSTSnapshotVersion *_lastRemoteSnapshotVersion;
+  // FSTSnapshotVersion *_lastRemoteSnapshotVersion;
```

Comment:
Commenting this code out doesn't seem right. I think you meant to just delete it.